src/test/java/given/a/spec/with/exception/in/aftereach/block/WhenRunningTheSpec.java

```diff
@@ -27,7 +27,8 @@ public void thereIsOneFailureForEachAffectedTest() throws Exception {
   @Test
   public void theFailureExplainsWhatHappened() throws Exception {
     assertThat(this.result.getFailures().get(0),
-        is(failure("a passing test", Fixture.SomeException.class, "kaboom")));
+        is(failure("a passing test", RuntimeException.class,
+            "given.a.spec.with.exception.in.aftereach.block.Fixture$SomeException: kaboom")));
```

Comment:
Changed purely down to the checked exception wrapping that now happens in `ThrowingConsumer`